services/app-web/src/pages/StateSubmission/RateDetails/RateDetails.test.tsx

```diff
@@ -1326,7 +1326,8 @@ describe('RateDetails', () => {
 
             expect(continueButton).toHaveAttribute('aria-disabled', 'true')
         })
-        it('disabled with alert when trying to continue while a file is still uploading', async () => {
+        // eslint-disable-next-line jest/no-disabled-tests
+        it.skip('disabled with alert when trying to continue while a file is still uploading', async () => {
```

Comment:
Skipped test is here - its for save as draft button behavior. It looked fine in manual QA 🤔

Reply:
this is working now